Makefile

```diff
@@ -1,20 +1,31 @@
 PRE_COMMIT_VERSION ?= 4.3.0
 
-.PHONY: init reinit check docs launch clean config
+.PHONY: init playwright test reinit check backup-database backup-file-storage backup restore-database restore-file-storage restore docs launch clean config
 
 init:
 	@test -d .venv || python3 -m venv .venv
 	.venv/bin/python -m pip install "pre-commit==$(PRE_COMMIT_VERSION)"
 	.venv/bin/pre-commit install
 
+playwright:
+	@test -d .venv || python3 -m venv .venv
+	.venv/bin/python -m pip install playwright pytest pytest-playwright pytest-order requests
+	.venv/bin/playwright install
+
+test:
+	DEBUG=pw:api .venv/bin/pytest --capture=no --exitfirst
+
+test-ui:
+	DEBUG=pw:api .venv/bin/pytest --headed --capture=no --exitfirst
+
 reinit:
 	rm -rf .venv
 	$(MAKE) init
 
 check:
 	.venv/bin/pre-commit run --all-files
 
-BACKUP_TIMESTAMP := $(shell date -u +%Y-%m-%d_%H-%M-%S_%Z)
+BACKUP_TIMESTAMP ?= $(shell date -u +%Y-%m-%d_%H-%M-%S_%Z)
```

Comment:
Nice! 👍 But maybe we should add something to the readme file about this added flexibility?

Reply:
Thinking about this, is there really a reason to allow changing the actual timestamp. Why would users like to create a timestamp different than the presence? Allowing them to change the format might be a good idea though

Reply:
Users might want to use a different format, so I think it's okay allowing to change it and still keeping this as default.

Reply:
In that case wouldn't it make more sense to expose a variable called BACKUP_TIMESTAMP_FORMAT?